documentation/source/coroutines.rst

```diff
@@ -53,22 +53,19 @@ Coroutines can :keyword:`return` a value, so that they can be used by other coro
 Concurrent Execution
 ====================
 
-Coroutines can be scheduled for concurrent execution with :func:`~cocotb.fork`, :func:`~cocotb.start`, and :func:`~cocotb.start_soon`.
+Coroutines can be scheduled for concurrent execution with :func:`~cocotb.start` and :func:`~cocotb.start_soon`.
+These concurrently running coroutines are called :type:`~cocotb.Task`\ s.
 
-:func:`~cocotb.fork` (deprecated) schedules and executes the new coroutine immediately,
-returning control to the calling task after the new coroutine finishes or yields control.
-No other pending tasks are run.
-
-The *async* function :func:`~cocotb.start` schedules the new coroutine to be executed concurrently,
+The *async* function :func:`~cocotb.start` schedules the coroutine to be executed concurrently,
 then yields control to allow the new task (and any other pending tasks) to run,
 before resuming the calling task.
 
-:func:`~cocotb.start_soon` schedules the new coroutine for future execution,
+:func:`~cocotb.start_soon` schedules the coroutine for future execution,
 after the calling task yields control.
 
 .. note::
-    The preferred way to schedule tasks is with :func:`~cocotb.start` and :func:`~cocotb.start_soon`.
-    :func:`~cocotb.fork` is deprecated and will be removed in a future version of cocotb.
+    ``cocotb.fork`` was a previously supported way of running coroutines concurrently,
```

Comment:
```suggestion
    In cocotb 1.x, ``cocotb.fork`` was typically used to run coroutines concurrently,
```

Relative time references don't age well in documentation, let's be specific.

Reply:
Given the other comment, maybe we should just remove this note.